2.13alpha - reviewer crashes on API 16

> As mentioned [here](https://stackoverflow.com/questions/8041537/how-to-reference-style-attributes-from-a-drawable/13471695#13471695) on API < 21 you can't use attrs to color in the XML drawable. So only way is to use a reference to a color resource (@color/YOURS_COLOR) or use #RGB format.

https://stackoverflow.com/a/34279007

```
08-28 15:55:22.552 2059-2059/com.ichi2.anki E/AndroidRuntime: FATAL EXCEPTION: main
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ichi2.anki/com.ichi2.anki.Reviewer}: android.view.InflateException: Binary XML file line #32: Error inflating class ImageView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2059)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084)
        at android.app.ActivityThread.access$600(ActivityThread.java:130)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:137)
        at android.app.ActivityThread.main(ActivityThread.java:4745)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:511)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
        at dalvik.system.NativeStart.main(Native Method)
     Caused by: android.view.InflateException: Binary XML file line #32: Error inflating class ImageView
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:830)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:736)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170)
        at com.ichi2.anki.AbstractFlashcardViewer.onCreate(AbstractFlashcardViewer.java:881)
        at com.ichi2.anki.Reviewer.onCreate(Reviewer.java:124)
        at android.app.Activity.performCreate(Activity.java:5008)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1079)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2023)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084) 
        at android.app.ActivityThread.access$600(ActivityThread.java:130) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:137) 
        at android.app.ActivityThread.main(ActivityThread.java:4745) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:511) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
        at dalvik.system.NativeStart.main(Native Method) 
     Caused by: android.content.res.Resources$NotFoundException: File res/drawable/preview_flashcard_hide_answer_background.xml from drawable resource ID #0x7f080128
        at android.content.res.Resources.loadDrawable(Resources.java:1923)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:601)
        at android.view.View.<init>(View.java:3336)
        at android.widget.ImageView.<init>(ImageView.java:114)
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:74)
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:69)
        at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:199)
        at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:119)
        at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1551)
        at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1602)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:675)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:746) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:830) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:736) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:489) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:396) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:352) 
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696) 
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170) 
        at com.ichi2.anki.AbstractFlashcardViewer.onCreate(AbstractFlashcardViewer.java:881) 
        at com.ichi2.anki.Reviewer.onCreate(Reviewer.java:124) 
        at android.app.Activity.performCreate(Activity.java:5008) 
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1079) 
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2023) 
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084) 
        at android.app.ActivityThread.access$600(ActivityThread.java:130) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:137) 
        at android.app.ActivityThread.main(ActivityThread.java:4745) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:511) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
        at dalvik.system.NativeStart.main(Native Method) 
     Caused by: java.lang.UnsupportedOperationException: Can't convert to color: type=0x2
        at android.content.res.TypedArray.getColor(TypedArray.java:326)
        at android.graphics.drawable.GradientDrawable.inflate(GradientDrawable.java:951)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:881)
        at android.graphics.drawable.StateListDrawable.inflate(StateListDrawable.java:183)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:881)
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:818)
        at android.content.res.Resources.loadDrawable(Resources.java:1920)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:601) 
        at android.view.View.<init>(View.java:3336) 
        at android.widget.ImageView.<init>(ImageView.java:114) 
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:74) 
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:69) 
        at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:199) 
        at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:119) 
        at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1551) 
        at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1602) 
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:675) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:746) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:830) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:736) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:489) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:396) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:352) 
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696) 
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170) 
        at com.ichi2.anki.AbstractFlashcardViewer.onCreate(AbstractFlashcardViewer.java:881) 
        at com.ichi2.anki.Reviewer.onCreate(Reviewer.java:124) 
        at android.app.Activity.performCreate(Activity.java:5008) 
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1079) 
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2023) 
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084) 
        at android.app.ActivityThread.access$600(ActivityThread.java:130) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:137) 
        at android.app.ActivityThread.main(ActivityThread.java:4745) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:511) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
        at dalvik.system.NativeStart.main(Native Method) 
```

Against current master:

```
E/AndroidRuntime: FATAL EXCEPTION: main
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ichi2.anki/com.ichi2.anki.Reviewer}: android.view.InflateException: Binary XML file line #41: Error inflating class ImageView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2059)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084)
        at android.app.ActivityThread.access$600(ActivityThread.java:130)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:137)
        at android.app.ActivityThread.main(ActivityThread.java:4745)
        at java.lang.reflect.Method.invokeNative(Native Method)
        at java.lang.reflect.Method.invoke(Method.java:511)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
        at dalvik.system.NativeStart.main(Native Method)
     Caused by: android.view.InflateException: Binary XML file line #41: Error inflating class ImageView
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:830)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:736)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170)
        at com.ichi2.anki.AbstractFlashcardViewer.onCreate(AbstractFlashcardViewer.java:880)
        at com.ichi2.anki.Reviewer.onCreate(Reviewer.java:124)
        at android.app.Activity.performCreate(Activity.java:5008)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1079)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2023)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084) 
        at android.app.ActivityThread.access$600(ActivityThread.java:130) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:137) 
        at android.app.ActivityThread.main(ActivityThread.java:4745) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:511) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
        at dalvik.system.NativeStart.main(Native Method) 
     Caused by: android.content.res.Resources$NotFoundException: File res/drawable/item_background_light_selectable_borderless.xml from drawable resource ID #0x7f0800f6
        at android.content.res.Resources.loadDrawable(Resources.java:1923)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:601)
        at android.view.View.<init>(View.java:3336)
        at android.widget.ImageView.<init>(ImageView.java:114)
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:74)
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:69)
        at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:199)
        at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:119)
        at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1551)
        at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1602)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:675)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:746) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:830) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:736) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:489) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:396) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:352) 
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696) 
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170) 
        at com.ichi2.anki.AbstractFlashcardViewer.onCreate(AbstractFlashcardViewer.java:880) 
        at com.ichi2.anki.Reviewer.onCreate(Reviewer.java:124) 
        at android.app.Activity.performCreate(Activity.java:5008) 
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1079) 
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2023) 
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084) 
        at android.app.ActivityThread.access$600(ActivityThread.java:130) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:137) 
        at android.app.ActivityThread.main(ActivityThread.java:4745) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:511) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
        at dalvik.system.NativeStart.main(Native Method) 
     Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #6: <item> tag requires a 'drawable' attribute or child tag defining a drawable
        at android.graphics.drawable.StateListDrawable.inflate(StateListDrawable.java:178)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:881)
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:818)
        at android.content.res.Resources.loadDrawable(Resources.java:1920)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:601) 
        at android.view.View.<init>(View.java:3336) 
        at android.widget.ImageView.<init>(ImageView.java:114) 
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:74) 
        at androidx.appcompat.widget.AppCompatImageView.<init>(AppCompatImageView.java:69) 
        at androidx.appcompat.app.AppCompatViewInflater.createImageView(AppCompatViewInflater.java:199) 
        at androidx.appcompat.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:119) 
        at androidx.appcompat.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1551) 
        at androidx.appcompat.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1602) 
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:675) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:746) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.parseInclude(LayoutInflater.java:830) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:736) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:749) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:489) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:396) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:352) 
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696) 
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170) 
        at com.ichi2.anki.AbstractFlashcardViewer.onCreate(AbstractFlashcardViewer.java:880) 
        at com.ichi2.anki.Reviewer.onCreate(Reviewer.java:124) 
        at android.app.Activity.performCreate(Activity.java:5008) 
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1079) 
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2023) 
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2084) 
        at android.app.ActivityThread.access$600(ActivityThread.java:130) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1195) 
        at android.os.Handler.dispatchMessage(Handler.java:99) 
        at android.os.Looper.loop(Looper.java:137) 
        at android.app.ActivityThread.main(ActivityThread.java:4745) 
        at java.lang.reflect.Method.invokeNative(Native Method) 
        at java.lang.reflect.Method.invoke(Method.java:511) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
        at dalvik.system.NativeStart.main(Native Method) 
```

--- a/AnkiDroid/src/main/java/com/ichi2/compat/CompatV16.java
+++ b/AnkiDroid/src/main/java/com/ichi2/compat/CompatV16.java
@@ -69,7 +69,7 @@ public class CompatV16 implements Compat {
         a.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
         final int fullscreenMode = Integer.parseInt(AnkiDroidApp.getSharedPrefs(a).getString("fullscreenMode", "0"));
         if (fullscreenMode >= FULLSCREEN_ALL_GONE) {
-            final LinearLayout answerButtons = a.findViewById(R.id.answer_options_layout);
+            final View answerButtons = a.findViewById(R.id.answer_options_layout);
             answerButtons.setVisibility(View.GONE);
         }
     }

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_again.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_again.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_red_700" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_again_dark.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_again_dark.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_red_800" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_all_black.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_all_black.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="#000000" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_all_plain.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_all_plain.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="#9E9E9E" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_easy.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_easy.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_light_blue_500" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_easy_dark.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_easy_dark.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_light_blue_800" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_good.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_good.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_green_500" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_good_dark.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_good_dark.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_green_800" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_hard.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_hard.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_blue_grey_700" />

--- a/AnkiDroid/src/main/res/drawable-v21/footer_button_hard_dark.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/footer_button_hard_dark.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <shape android:shape="rectangle" >
             <solid android:color="@color/material_blue_grey_800" />

--- a/AnkiDroid/src/main/res/drawable-v21/item_background_light_selectable.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/item_background_light_selectable.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?attr/answerButtonsHighlightColor">
+    android:color="@color/answer_buttons_highlight_color">
     <item android:id="@android:id/mask">
         <color android:color="@android:color/white" />
     </item>

--- a/AnkiDroid/src/main/res/drawable-v21/item_background_light_selectable_borderless.xml
+++ b/AnkiDroid/src/main/res/drawable-v21/item_background_light_selectable_borderless.xml
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="?attr/answerButtonsHighlightColor" />
+    android:color="@color/answer_buttons_highlight_color" />

--- a/AnkiDroid/src/main/res/drawable/item_background_light_selectable.xml
+++ b/AnkiDroid/src/main/res/drawable/item_background_light_selectable.xml
@@ -3,10 +3,10 @@
     <item android:state_window_focused="false" android:drawable="@android:color/transparent" />
 
     <!-- Even though these two point to the same resource, have two states so the drawable will invalidate itself when coming out of pressed state. -->
-    <item android:state_focused="true"  android:state_enabled="false" android:state_pressed="true" android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="true"  android:state_enabled="false"                              android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="true"                                android:state_pressed="true" android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="false"                               android:state_pressed="true" android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="true"                                                             android:drawable="?attr/answerButtonsHighlightColor" />
+    <item android:state_focused="true"  android:state_enabled="false" android:state_pressed="true" android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="true"  android:state_enabled="false"                              android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="true"                                android:state_pressed="true" android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="false"                               android:state_pressed="true" android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="true"                                                             android:drawable="@color/answer_buttons_highlight_color" />
     <item android:drawable="@android:color/transparent" />
 </selector>

--- a/AnkiDroid/src/main/res/drawable/item_background_light_selectable_borderless.xml
+++ b/AnkiDroid/src/main/res/drawable/item_background_light_selectable_borderless.xml
@@ -3,10 +3,10 @@
     <item android:state_window_focused="false" android:drawable="@android:color/transparent" />
 
     <!-- Even though these two point to the same resource, have two states so the drawable will invalidate itself when coming out of pressed state. -->
-    <item android:state_focused="true"  android:state_enabled="false" android:state_pressed="true" android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="true"  android:state_enabled="false"                              android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="true"                                android:state_pressed="true" android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="false"                               android:state_pressed="true" android:drawable="?attr/answerButtonsHighlightColor" />
-    <item android:state_focused="true"                                                             android:drawable="?attr/answerButtonsHighlightColor" />
+    <item android:state_focused="true"  android:state_enabled="false" android:state_pressed="true" android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="true"  android:state_enabled="false"                              android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="true"                                android:state_pressed="true" android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="false"                               android:state_pressed="true" android:drawable="@color/answer_buttons_highlight_color" />
+    <item android:state_focused="true"                                                             android:drawable="@color/answer_buttons_highlight_color" />
     <item android:drawable="@android:color/transparent" />
 </selector>

--- a/AnkiDroid/src/main/res/values/attrs.xml
+++ b/AnkiDroid/src/main/res/values/attrs.xml
@@ -66,7 +66,6 @@
     <attr name="hardButtonTextColor" format="color"/>
     <attr name="goodButtonTextColor" format="color"/>
     <attr name="easyButtonTextColor" format="color"/>
-    <attr name="answerButtonsHighlightColor" format="color"/>
     <!-- Card Browser Colors -->
     <attr name="cardBrowserDivider" format="color"/>
     <!-- Stats colors -->

--- a/AnkiDroid/src/main/res/values/colors.xml
+++ b/AnkiDroid/src/main/res/values/colors.xml
@@ -31,6 +31,8 @@
     <color name="white">#ffffff</color>
     <color name="black">#000000</color>
 
+    <color name="answer_buttons_highlight_color">#66FFFFFF</color>
+
 
     <!-- Material Design: http://www.google.com/design/spec/style/color.html#color-color-palette -->
     <!-- Can eventually be shortened once settled on a color scheme -->

--- a/AnkiDroid/src/main/res/values/theme_dark.xml
+++ b/AnkiDroid/src/main/res/values/theme_dark.xml
@@ -65,7 +65,6 @@
         <item name="hardButtonRef">@drawable/footer_button_hard_dark</item>
         <item name="goodButtonRef">@drawable/footer_button_good_dark</item>
         <item name="easyButtonRef">@drawable/footer_button_easy_dark</item>
-        <item name="answerButtonsHighlightColor">#66FFFFFF</item>
         <!-- Card Browser Colors -->
         <item name="cardBrowserDivider">@color/white</item>
         <!-- Stats colors -->

--- a/AnkiDroid/src/main/res/values/theme_light.xml
+++ b/AnkiDroid/src/main/res/values/theme_light.xml
@@ -73,7 +73,6 @@ APIs. It's visible when there aren't enough decks to fill the screen.
         <item name="hardButtonTextColor">@color/white</item>
         <item name="goodButtonTextColor">@color/white</item>
         <item name="easyButtonTextColor">@color/white</item>
-        <item name="answerButtonsHighlightColor">#AAFFFFFF</item>
         <!-- Reviewer button drawables -->
         <item name="againButtonRef">@drawable/footer_button_again</item>
         <item name="hardButtonRef">@drawable/footer_button_hard</item>

